.repo-metadata.json

```
@@ -2,14 +2,15 @@
     "name": "pandas-gbq",
     "name_pretty": "Google BigQuery connector for pandas",
     "product_documentation": "https://cloud.google.com/bigquery",
-    "client_documentation": "https://pandas-gbq.readthedocs.io/en/latest/",
+    "client_documentation": "https://cloud.google.com/python/docs/reference/pandas-gbq/latest",
```

`pandas-gbq` isn't under Cloud RAD because it's not a client library (also needs intersphinx to work well)

```suggestion
    "client_documentation": "https://googleapis.dev/python/pandas-gbq/latest/",
```